### ISSUE TITLE
SDFGI - Probes all black, no lights reflected in shadow, shadows pitch black

### ISSUE THREAD

**Issue by sietse85** [godotengine/godot#82803]
### Godot version

4.1.1.stable.mono.official

### System information

Linux Mint  (GTX 1060 - NVidia driver 535)

### Issue description

The probes from SDFGI stay black and don't reflect any light. No light seems to be reflected.
![image](https://github.com/godotengine/godot/assets/10598991/59ae8a30-6713-4be7-926f-57076ef8a0d1)

Found an old similar issue here:
https://github.com/godotengine/godot/issues/39969

### Steps to reproduce

Open the minimal project in the editor and toggle on SDFGI probes. Also notice the pitch black shadows, and notice no light is beeing reflected by moving inside the house in the scene.

### Minimal reproduction project

Dropbox link:
https://www.dropbox.com/scl/fi/mro6cf8aukwmuglk7c9bq/quasargame.zip?rlkey=bo8ck0nt0vq7q7xkfar3ugcby&dl=0

**Comment by bitsawer** [comment]
After tweaking some settings, looks like you need to enable depth-prepass in project settings for the SDFGI to work. Not sure if this is a bug or intentional design requirement that should be documented.

Also, the project in your dropbox includes both .git and .godot folders that are several hundred megabytes in size. You don't need to include them in the .zip file. After that, you the .zip is small enough so you can attach it to a Github post.

**Comment by sietse85** [comment]
@bitsawer Enabling depth-pre pass made it work. Thanks! 

b.t.w. including the .git was unintentional. I will keep that in mind next time.

**Comment by Zireael07** [comment]
I think the need to enable depth pre-pass should be documented.

**Comment by Calinou** [comment]
Reopening, as this needs to be documented or investigated (if this is a bug).

**Comment by sietse85** [comment]
Yes it would be nice if this would be mentioned at least in the SDFGI documentation on the Godot Docs website.
